api/v1beta1/vmprobe_types.go

```diff
@@ -89,9 +89,10 @@ type VMProberSpec struct {
 type VMProbeStatus struct {
 }
 
+//  VMProbe defines a probe for targets, that will be executed with prober,
```

Comment:
could you please clarify what it's and why it exists (its mission) 
e.g VMProbe does something and existis to solve problem A